--- idf_tools.py	2021-01-01 02:29:19.850422647 +0100
+++ ../../idf_tools.py	2021-01-01 02:28:38.522695701 +0100
@@ -98,6 +98,7 @@
 PLATFORM_LINUX64 = 'linux-amd64'
 PLATFORM_LINUX_ARM32 = 'linux-armel'
 PLATFORM_LINUX_ARM64 = 'linux-arm64'
+PLATFORM_LINUX_PPC64LE = "linux-ppc64le"
 
 
 # Mappings from various other names these platforms are known as, to the identifiers above.
@@ -131,6 +132,8 @@
     'Linux-arm64': PLATFORM_LINUX_ARM64,
     'Linux-aarch64': PLATFORM_LINUX_ARM64,
     'Linux-armv8l': PLATFORM_LINUX_ARM64,
+    PLATFORM_LINUX_PPC64LE: PLATFORM_LINUX_PPC64LE,
+    'Linux-ppc64le': PLATFORM_LINUX_PPC64LE,
 }
 
 UNKNOWN_PLATFORM = 'unknown'
